### DISCUSSION TITLE
Save soil datasets

### DISCUSSION THREAD

**Discussion by ellenalamont17** [earthlab-education/Earth-Analytics-2023-01-Intro#675] [! GitHub help]
Hi everyone, 

I know I should know how to do this, but I just can't seem to do this correctly. I am currently trying to save my soil data to my directory so that I have a copy to work from.  I had no issues obtaining a URL and actually opening the data with rasterio from the URL, but how do I actually save those files? For things like  APPEEARS and EarthExplorer there was a downloader that downloaded them to the specified directory. But what if I am not using one of those fancy downloader services?  I know this must be a simple answer, but at this point, I am other thinking everything. Thanks for your help!

Example of  current code:
``````
# Define the data url
butte_url = someurl.tif

# Open url to work with array
rxr.open_rasterio(butte_url, masked=True).squeeze()
``````

**Comment by NobodyLivesHere** [comment]
Here is what i have been using.  Raster_dir is just the folder where I wanted to keep the download directories in and polaris_url is just the file url amended with lat and lon derived from the bounds of the grassland gdf.

![image](https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/88915377/d032b647-b86c-4ee8-a508-a6c5476b9ae5)
